megazords/ios/MozillaAppServices.xcodeproj/project.pbxproj

```diff
@@ -279,9 +287,12 @@
 			children = (
 				394A807525EE951D00FAF26F /* Generated */,
 				394A807325EE94C600FAF26F /* nimbus.udl */,
+				394A809725F94E1300FAF26F /* Nimbus.swift */,
+				395992B725FBE40300E3185F /* NimbusApi.swift */,
+				394D6C612600E64E008F9CE5 /* NimbusCreate.swift */,
 			);
 			name = Nimbus;
-			path = "../../components/external/nimbus-sdk/nimbus";
+			path = ../../components/nimbus/ios/Nimbus;
```

Comment:
Does the path to the `nimbus.udl` child of this group need to be updated as a result of this change? It looks like the file entry for `nimbus.udl` has its path configured to be relative to its containing group, and this changes the path of its containing group.